include/cantera/kinetics/Arrhenius.h

```diff
@@ -203,7 +203,7 @@ class Arrhenius3 : public ArrheniusBase
  * activation energy for electron is included.
  *
  *   \f[
- *        k_f =  A T_e^b \exp (-E_a/RT) \exp (-E_{a,e}/RT_e)
+ *        k_f =  A T_e^b \exp (-E_{a,gas}/RT) \exp (E_{a,electron} (T_e - T)/(R T T_e))
```

Comment:
Should this be just `E_{a,e}` rather than `E_{a,electron}`, as you've done in the Python documentation?